### PR TITLE
DROTH-746_Change_infotext_on_bus_stop_direction_creation

### PR DESCRIPTION
**Pull request by oskarmalmstrom**
first checks if road is one directional, then if the stop direction is the same as the road direction

### CODE REVIEW

**Comment by joanacoelho** [comment]
looks good

File: UI/src/model/SelectedMassTransitStop.js [hunk 1]
```diff
@@ -277,6 +277,21 @@
...
+        var stopDirection = get('validityDirection');
+
+        if(roadLinkDirection != 1)
+          return stopDirection != roadLinkDirection;
+        return true;
+      }else{
+        return false;
+      }
+    };
+
```
Comment (MAC-CGI) [inline UI/src/model/SelectedMassTransitStop.js]: validateDirectionsForSave and validateDirectionsForCreation have a similar code only with a different return.
Could you probably create a function to return the RoadLinkDirection. Something like that:

var getRoadLinkDirection = function(){
      var nearestLine = geometrycalculator.findNearestLine(roadCollection.getRoadsForMassTransitStops(), currentAsset.payload.lon, currentAsset.payload.lat);
      var linkId = nearestLine.linkId;
      if (!currentAsset.linkId)
        currentAsset.linkId = linkId;
      var directions_decode = {BothDirections: 1, TowardsDigitizing: 2, AgainstDigitizing: 3};
      return directions_decode[nearestLine.trafficDirection];
    };

**Comment by halied** [comment]
Ok

**Comment by marquesfm** [comment]
Ok for me

**Pull request merged**